pkg/config/datafileprojectconfig/mappers/feature.go

```diff
@@ -40,6 +41,7 @@ func MapFeatures(featureFlags []datafileEntities.FeatureFlag, rolloutMap map[str
 			if experiment, ok := experimentMap[experimentID]; ok {
 				featureExperiments = append(featureExperiments, experiment)
 			}
+			experimentFeatureMap[experimentID] = append(experimentFeatureMap[experimentID], featureFlag.ID)
```

Comment:
Instead of doing this, we can simplify and add an extra bool property to `entities.Experiment` called `IsFeatureExperiment` so then later on we can just check for this property instead of having to compute it from the ProjectConfig.